components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -40,35 +49,64 @@ export class LayoutFacadeService {
       (notifications: Notification[]) => {
         this.showLicenseNotification =
           notifications &&  notifications.some(n => n.type === Type.license);
-        this.updateContentHeight(
-          this.showLicenseNotification ? Height.Banner : Height.Navigation);
+          if (this.showLicenseNotification) {
+            this.totalElementsHeights += Height.Banner;
+            this.updateContentHeight();
+          } else {
+            this.totalElementsHeights -= Height.Banner;
+            this.updateContentHeight();
+          }
+      });
+
+    this.store.select(allProjects).pipe(
+        takeUntil(this.isDestroyed)
+      ).subscribe((projectList: Project[]) => {
+        if(projectList.some(p => p.status === 'EDITS_PENDING')) {
+          this.totalElementsHeights += Height.PendingEditsBar;
+          this.updateContentHeight();
+        }
       });
+      this.totalElementsHeights += Height.Navigation;
+      this.updateContentHeight();
+  }
+
+  ngOnDestroy(): void {
+    this.isDestroyed.next(true);
+    this.isDestroyed.complete();
+  }
+
+  /* 
+    Pass true for fullPage
+  */
```

Comment:
[supernit] Not sure this comment is necessary. If this behaved otherwise I'd be quite surprised! Fine with it staying though no worries either way.

Reply:
@tylercloke you're right, it works as expected. I'll remove comment 👍🏻